### ISSUE TITLE
Inability to add a horizontal buttons stack as header instead of footer

### ISSUE THREAD

**Issue by sayam93** [Clooos/Bubble-Card#418]
![RPReplay_Final1711318750](https://github.com/Clooos/Bubble-Card/assets/163408168/679500c0-0316-49f7-85c8-d8592e898d9d)
Is there any way to use the horizontal buttons stack as a header (or part of the body) instead of footer?  If not, having that option would be really useful! Am currently using button card with swipe card to achieve this.

**Comment by kylewhirl** [comment]
I agree. How did you do this? Having the horizontal button stack at the bottom is completely useless for me on iphone

**Comment by sayam93** [comment]
> I agree. How did you do this? Having the horizontal button stack at the bottom is completely useless for me on iphone

My latest iteration uses a paper buttons row. Let me know if you need any help.
